src/traversal.js

```diff
@@ -1,4 +1,4 @@
-import { base58btc } from 'multiformats/bases/base58'
+import { base58btc } from './bases/base58.js'
```

Comment:
This should have originally used the relative pathing like all other src/ modules. It's probably one reason why @Gozala made tsconfig.json verbose with mapping because we do this through test/ but avoid it here, mostly. So I'm just applying consistency here.

Reply:
> It's probably one reason why @Gozala made tsconfig.json verbose with mapping because we do this through test/ but avoid it here, mostly. So I'm just applying consistency here.

I think I had to do it for tests and not this.